sdk/messaging/azservicebus/example_shared_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azservicebus_test
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus"
+	"github.com/joho/godotenv"
+)
+
+// Helper functions to make the main examples compilable
+
+func exitOnError(message string, err error) {
+	// these errors/failures are expected since the example
+	// code, at the moment, can't run.
+	if err == nil {
+		return
+	}
+
+	log.Printf("(error in example): %s: %s", message, err.Error())
+}
+
+func yourLogicForProcessing(message *azservicebus.ReceivedMessage) {}
+
+func yourMessageHandler(message *azservicebus.ReceivedMessage) error {
+	return nil
+}
+
+func yourErrorHandler(err error) {}
+
+var connectionString string
+var queueName string
+var topicName string
+var subscriptionName string
+var client *azservicebus.Client
+var sender *azservicebus.Sender
+var receiver *azservicebus.Receiver
+var messages []*azservicebus.ReceivedMessage
+var err error
```

Comment:
I think we should get rid of all of these as well.

Reply:
They're there so the individual example tests are succinct. Otherwise each one of them needs to have the boilerplate or they're not individually compileable.